consumers/writers/influxdb/README.md

```diff
@@ -12,7 +12,7 @@ default values.
 | ----------------------------- | --------------------------------------------------------------------------------- | ---------------------- |
```

Comment:
Add docs for 

- MF_INFLUXDB_HOST
- MF_INFLUXDB_PROTOCOL
- MF_INFLUXDB_TIMEOUT
- MF_INFLUXDB_ORG
- MF_INFLUXDB_BUCKET
- MF_INFLUXDB_TOKEN
- MF_INFLUXDB_HTTP_ENABLED
- MF_INFLUXDB_INIT_MODE

Reply:
Also here https://github.com/AryanGodara/mainflux/blob/influx-db-update/consumers/writers/influxdb/README.md?plain=1#L41